osf/models/user.py

```diff
@@ -746,6 +748,8 @@ def save(self, *args, **kwargs):
             self.update_search()
             # TODO
             # self.update_search_nodes_contributors()
+        if first_save:
```

Comment:
If you did this in a signal instead of in save you'd get a `created` function param telling you that it was created. Check the bottom of `osf/models/node.py` for an example.